package/kedro_viz/integrations/deployment/aws_deployer.py

```diff
@@ -0,0 +1,35 @@
+"""`kedro_viz.integrations.deployment.aws_deployer` defines
+deployment class for AWS S3"""
+
+import logging
+
+import fsspec
+
+from kedro_viz import __version__
+from kedro_viz.integrations.deployment.base_deployer import BaseDeployer
+
+_S3_PROTOCOL = "s3"
```

Comment:
Does this need to be a global?

Reply:
not really, but I left it as is from earlier PRs. Though we did not use the global variable in multiple functions, it is used in multiple places in the file. Also, if the protocol changes, it is easy to maintain. Do you see any issue in using a global here ?